src/AltinnCore/Common/Services/Implementation/ArchiveStudioSI.cs

```diff
@@ -13,7 +13,7 @@
 namespace AltinnCore.Common.Services.Implementation
 {
     /// <summary>
-    /// Implementation for archive service
+    /// Implementation for archive
```

Comment:
I disagree with these comment changes. This actually is a service, and is not the term we are looking to change in this issue.

Reply:
I agree, this is a service, but not a Altinn Service, but a system service (SI stands for Service Implementation)